pkg/cri/cri.go

```diff
@@ -97,7 +99,13 @@ func initCRIService(ic *plugin.InitContext) (interface{}, error) {
 		return nil, fmt.Errorf("failed to create containerd client: %w", err)
 	}
 
-	s, err := server.NewCRIService(c, client)
+	var s server.CRIService
+	if os.Getenv("ENABLE_CRI_SANDBOXES") != "" {
+		log.G(ctx).Warn("using experimental CRI Sandbox server")
+		s, err = sbserver.NewCRIService(c, client)
+	} else {
+		s, err = server.NewCRIService(c, client)
```

Comment:
consider doing an info log in or for both modes regarding the env variable option(s)...

Reply:
Added https://github.com/containerd/containerd/pull/7164/commits/98a1b7ff1b53c847506a93673398d53ba86761d4